test/test_dispatch.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from pyro_api import handlers, infer, pyro, pyro_backend
+from pyro_api.testing import MODELS
+
+pytestmark = pytest.mark.stage('unit')
+
+
+@pytest.mark.filterwarnings("ignore", category=UserWarning)
+@pytest.mark.parametrize('model', MODELS)
+@pytest.mark.parametrize('backend', ['pyro', 'numpy'])
+def test_mcmc_interface(model, backend):
```

Comment:
Just a reminder that any code in this file is not distributed in the package, and therefore is not usable by other libraries. Issue number 1 should be "move all testing code into the main library".

Reply:
Done.